src/main/java/org/apache/commons/daemon/support/DaemonWrapper.java

```diff
@@ -40,12 +40,17 @@
     private final static String STOP_CLASS      = "stop";
     private final static String STOP_METHOD     = "stop.method";
     private final static String STOP_ARGS       = "stop.args";
-    private String              configFileName  = null;
+    private String              configFileName;
     private final DaemonConfiguration config;
 
     private final Invoker             startup;
     private final Invoker             shutdown;
 
+    /**
+     * An empty immutable {@code String} array.
```

Comment:
This does not need to be public does it?

Reply:
Changed